site/content/contribute/server/developer-setup/osx.md

```diff
@@ -15,38 +15,38 @@
     If you don't migrate your data, the new, docker-compose-managed containers will start out empty. To remove the old containers -- destroying any existing data -- use `make clean-old-docker`.
      
     
-2. Download and install homebrew: https://brew.sh/.
+3. Download and install homebrew: https://brew.sh/.
 
-3. Install Go:
+4. Install Go:
     ```sh
     brew install go
```

Comment:
@lieut-data While we're at it, should we:
```suggestion
     brew install go@1.12
```

Reply:
I don't oppose the change, but I'd probably recommend our fixing that separately since I expect it's a bunch of research to determine how to install that version on other platforms.

Reply:
Looks like it's Arch and Windows (non wsl) that don't have 1.12 specific. I'm fine for a separate PR for it, it was just a thought.